telegram/message.py

```diff
@@ -106,6 +106,7 @@ class Message(TelegramObject):
         passport_data (:class:`telegram.PassportData`): Optional. Telegram Passport data.
         poll (:class:`telegram.Poll`): Optional. Message is a native poll,
             information about the poll.
+        dice (:class:`telegram.Dice`): Optional. Message is a dice with random value from 1 to 6.
```

Comment:
Attributes are supposed to be shorter

```suggestion
        dice (:class:`telegram.Dice`): Optional. Message is a dice.
```

Reply:
changed